docs/4.4/api-reference.md

```diff
@@ -11,57 +11,328 @@ Teleport is currently working on documenting our API.
 
         We are currently working on this project. If you have an API suggestion, [please complete our survey](https://docs.google.com/forms/d/1HPQu5Asg3lR0cu5crnLDhlvovGpFVIIbDMRvqclPhQg/edit).
 
+# gRPC APIs
+
+In most cases, you can interact with teleport using our cli tools, [tsh](cli-docs.md#tsh) and [tctl](cli-docs.md#tctl). However, there are some scenarios where you may need to interact with teleport programmatically. For this purpose, the same gRPC auth API that `tctl` and `tsh` use is available for direct usage.
+
+## Go Examples
+
+!!! Note
+    The Go examples depend on some code that won't be released until `Teleport version 5.0`. Until then, it would be best to start experimenting with the API with the latest [master branch of Teleport](https://github.com/gravitational/teleport).
```

Comment:
I don't think you need the code blocks around "Teleport version 5.0", also you can just say Teleport 5.0.